CHANGELOG.rst

```diff
@@ -1,40 +1,49 @@
 2.9.0.dev
 =========
 
-New Features
-------------
+**New Features**
```

Comment:
That didn't work... that only works for explicit hyperlink targets, not implicit ones such as those generated by section titles. [The docs](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#implicit-hyperlink-targets) don't mention any way to avoid generating the links.

Any other ideas? :confused: